example/components/widgets/NodeSelection.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, Button } from "@material-ui/core";
+import { DiagramEngine, PortWidget } from "@projectstorm/react-diagrams";
+import * as React from "react";
+
+type styleObject = {
+  background: string;
+  borderColor: string;
+  boxShadow: string;
+}
+
+export interface NodeSelectionProps {
+  style: styleObject;
+  engine: DiagramEngine;
+  port: any;
+}
+
+class NodeSelection extends React.Component<NodeSelectionProps> {
+  render() {
+    const { style, engine, port } = this.props;
+    return (
+      <>
+        <Button className="node-button">
```

Comment:
@afonsobspinto @vidhya-metacell I would like to see this refactored , this is not so DRY (see https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)